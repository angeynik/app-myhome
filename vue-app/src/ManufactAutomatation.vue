<template>
    <div>
      <nav>
        <router-link to="/dashboard">Dashboard</router-link> |
        <router-link to="/smart-home">Smart Home</router-link> |
        <router-link to="/manufact-automatation">Manufact Automation</router-link>
      </nav>

      <h1>Manufact Automation</h1>
      <p>Welcome to the Manufact Automation page!</p>
    </div>
  </template>
  
  <script>
  export default {
    name: 'ManufactAutomatation',
  };
  </script>
 
  <style scoped>
  /* Стили для ManufactAutomatation */
  </style>