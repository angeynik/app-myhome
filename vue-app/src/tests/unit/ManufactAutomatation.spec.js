import { mount } from '@vue/test-utils';
import ManufactAutomatation from '@/ManufactAutomatation.vue';

describe('ManufactAutomatation.vue', () => {
  it('matches the snapshot', () => {
    const wrapper = mount(ManufactAutomatation);
    expect(wrapper.html()).toMatchSnapshot();
  });
});