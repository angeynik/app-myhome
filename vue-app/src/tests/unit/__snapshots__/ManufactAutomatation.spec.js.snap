// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ManufactAutomatation.vue matches the snapshot 1`] = `
"<div>
  <div>
    <nav>
      <router-link-stub to="/login"></router-link-stub> | <router-link-stub to="/dashboard"></router-link-stub> | <router-link-stub to="/smart-home"></router-link-stub> | <router-link-stub to="/manufact-automatation"></router-link-stub> |
      <!-- Profile (доступна для level 2 и 3) -->
      <!--v-if-->
      <!-- Users (доступна для level 3) -->
      <!--v-if-->
    </nav>
  </div>
  <p style="width: 100%; height: 1px;"></p>
  <div>
    <h1>Manufact Automation</h1>
    <p>Welcome to the Manufact Automation page!</p>
  </div>
</div>"
`;
